File: seava.mod.bd.ui.extjs/src/main/resources/webapp/seava/bd/ui/extjs/ds/CurrencyXRateProviderLov_Ds.js
/**
 * Solutii Ecommerce, Automatizare, Validare si Analiza | Seava.ro
 * Copyright: 2013 Nan21 Electronics SRL. All rights reserved.
 * Use is subject to license terms.
 */
	
Ext.define("seava.bd.ui.extjs.ds.CurrencyXRateProviderLov_Ds", {
	extend: 'Ext.data.Model',
	
	fields: [
		{name:"id", type:"string"},
		{name:"clientId", type:"string"},
		{name:"code", type:"string"},
		{name:"name", type:"string"},
		{name:"description", type:"string"},
		{name:"active", type:"boolean"},
		{name:"refid", type:"string"}
	]
});

File: seava.mod.bd.presenter/src/main/java/seava/bd/presenter/impl/currency/model/CurrencyXRateProviderLov_Ds.java
/**
 * Solutii Ecommerce, Automatizare, Validare si Analiza | Seava.ro
 * Copyright: 2013 Nan21 Electronics SRL. All rights reserved.
 * Use is subject to license terms.
 */
package seava.bd.presenter.impl.currency.model;

import seava.bd.domain.impl.currency.CurrencyXRateProvider;
import seava.j4e.api.annotation.Ds;
import seava.j4e.api.annotation.SortField;
import seava.j4e.presenter.impl.model.AbstractTypeWithCodeLov_Ds;

@Ds(entity = CurrencyXRateProvider.class, sort = {@SortField(field = CurrencyXRateProviderLov_Ds.f_code)})
public class CurrencyXRateProviderLov_Ds
		extends
			AbstractTypeWithCodeLov_Ds<CurrencyXRateProvider> {

	public CurrencyXRateProviderLov_Ds() {
		super();
	}

	public CurrencyXRateProviderLov_Ds(CurrencyXRateProvider e) {
		super(e);
	}
}
